modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can add, delete, and deprecate API versions, also known as _Exchange assets_, in Anypoint Exchange (Exchange). An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information.
+In Anypoint Exchange (Exchange), you can add, delete, and deprecate API versions, also known as _Exchange assets_. An API version in Exchange can have its own RAML file, API portal, and URL. See xref:exchange::manage-versions.adoc[Manage Versions] for more information.
```

Comment:
The way you construct the more information text should be consistent. Here you start it with the reference (See xref:exchange::manage-versions.adoc[Manage Versions] for more information.) Later on, you start it with "For more information on..."

It is better to start it with "For more information about..." because users can decide more quickly if they want to look at the reference. Kim wanted us to use "about" instead of "on."